ethereum-common/src/beacon/light.rs

```diff
@@ -107,8 +107,13 @@ pub struct Block {
     pub body: BlockBody,
 }
 
-impl From<super::Block> for Block {
-    fn from(value: super::Block) -> Self {
+impl<BlockBody, BlockBodyLight> From<beacon::block::BlockGeneric<BlockBody>>
```

Comment:
My personal take (obviously, subjective). Long names like `BlockBody` or `BlockBodyLight` sound like type or trait names whereas in fact here they are generic type parameters. If it was `BB: TreeHash` and `LBB: From<BB> + TreeHash` for type variables I personally would have easier time digesting it.